### ISSUE TITLE
Call logs, sms, wifi password

### ISSUE THREAD

**Issue by gericom-hummer** [jensstein/oandbackup#95]
I think there should have also the possibility of blacking up also the SMS, WiFi passwords and call logs, like TiB

**Comment by jensstein** [comment]
sms messages are in mmssms.db of com.android.providers.telephony, call logs are in contacts2.db of com.android.providers.contacts and wifi password are in wpa_supplicant.conf of the special wifi.access.points package.
i don't think i would want to handle them in a special way outside of these packages. but i have thought about adding a paragraph to the readme and the in-app help notes about it. i just forgot about it again.

**Comment by gericom-hummer** [comment]
Thanx, i'll make periodically backups to theese.

**Comment by fajabird** [comment]
@jensstein 
please have a look at this great open-source project: https://github.com/handschuh/Slight-backup
It hast some very easy routines to backup call-log, SMS, dictionary, playlists, settings and even contacts. I would like to see them (call-log first) in oandbackup, so I can have all in one app, PLUS schedules which I still miss in SlightBackup :-)

**Comment by jensstein** [comment]
yes, i am aware of that program.
but what would the value of having another set of operations for these things (call logs, sms, etc) be in your opinion when they can be backed up as descriped above (you can back up com.android.providers.contacts to get your call logs for example)?
are there problems or bugs with the current way that i haven't realised?

**Comment by DJaeger** [comment]
I think a difference in the way of doing the backup and restore could be that by backing up and restoring the provider application will overwrite all existing content and *walking through* would add them.
Or am I wrong?
This could be an option on restoring.

**Comment by fajabird** [comment]
Yes - thats one of my points. Another is that moving android specific files between different android versions (this is my default use case for doing a backup and restoring it) could cause issues, too. But I understand that there is already a method implemented and as long it works for most people its low priority to change that.

**Comment by DJaeger** [comment]
That's a much better point, you are right.

**Comment by gitgt** [comment]
For Wi-Fi passwords, F-droid has WiFi Key View.

**Comment by ildar** [comment]
Also for SMS and call logs there's SMS backup+
J FYI

**Comment by DJaeger** [comment]
WiFi Key View doesn't allow to automatically backup and with SMS backup + you can not save all your backups in one place.

**Comment by Schroedingers-Cat** [comment]
I have never succeeded in restoring text messages, call logs etc. from both, oandbackup and manual file backup. As soon as I restore the specified apps from oandbackup or copy the files and set the correct permissions manually, I get spammed with "Dailer as stopped working" dialogs. Is there an official and working way to backup this?

**Comment by duttyend** [comment]
> sms messages are in mmssms.db of com.android.providers.telephony, call logs are in contacts2.db of com.android.providers.contacts and wifi password are in wpa_supplicant.conf of the special wifi.access.points package.
> i don't think i would want to handle them in a special way outside of these packages. but i have thought about adding a paragraph to the readme and the in-app help notes about it. i just forgot about it again.

Dear @jensstein ,

Before all, thank you so much for your awesome oandbackup !!!

As others have said, it would be great to be able to backup / restore SMS and call log from oandbackup, but I understand your point.

As an information for people who are interested in doing a backup/restore on recent Android versions, the new path is "/data/data/com.android.messaging" and the new file is "bugle_db".

Have a great day!
